app/models/room.rb

```diff
@@ -40,6 +42,16 @@ def number_with_type
     "#{number} (#{type})"
   end
 
+  # Sets the original suite to the current suite, unless it is already set
+  #
+  # @params [Hash] attrs Other attributes to update
+  #
+  # @return [Boolean] Whether or not the update was successful
+  def store_original_suite!(**attrs)
```

Comment:
We want the original suite to always be the first suite the rooms belongs into, so once this attribute is set it shouldn't change.

In that scenario, `original_suite` would always be the room's *first* suite, not any of the ones that it belonged to during previous merges.